docs/4.2/features/ssh_pam.md

```diff
@@ -195,22 +199,31 @@ I've updated this to provide a message to users of Teleport, so they know they a
 being audited. 
 
 ```bash
-$ nano /etc/motd
```

Comment:
I tried to do this as an example that I changed the file, it might not work well in this example. I think just showing cat is good.

Reply:
Yeah, I get it for sure - I just always try to make anything that runs a command have 'authentic' output after it so that people know roughly what to expect.